client/checkout/classic/payment-processing.js

```diff
@@ -400,6 +401,27 @@ export async function mountStripePaymentElement( api, domElement ) {
 		gatewayUPEComponents[ paymentMethodType ].upeElement ||
 		( await createStripePaymentElement( api, paymentMethodType ) );
 	upeElement.mount( domElement );
+	upeElement.on( 'change', ( e ) => {
+		gatewayUPEComponents[ paymentMethodType ].isPaymentInformationComplete =
+			e.complete;
+	} );
+	upeElement.on( 'loaderror', ( e ) => {
```

Comment:
```suggestion
	upeElement.on( 'change', ( event ) => {
		gatewayUPEComponents[ paymentMethodType ].isPaymentInformationComplete =
			event.complete;
	} );
	upeElement.on( 'loaderror', ( event ) => {
```
[Just sayin'.](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions) 🤷

Reply:
Unfortunately, `event` is already defined a few lines above, so I can't name this the same :( 

https://github.com/Automattic/woocommerce-payments/blob/feat/error-notice-on-1m-amount/client/checkout/classic/payment-processing.js#L392